spec/serializers/web_hook_post_serializer_spec.rb

```diff
@@ -33,6 +33,7 @@ def serialized_for_user(u)
       :primary_group_name,
       :flair_name,
       :flair_group_id,
+      :badges_granted,
```

Comment:
I don't think we need to include the `badges_granted` attribute in the `web_hook_post_serializer` as it isn't very useful for webhooks. You can exclude it by adding the key to [this array](https://github.com/discourse/discourse/blob/72132c35fb3f90b0323c6ce735e4e7bd4e783100/app/serializers/web_hook_post_serializer.rb#L18-L33).